portfolio/src/main/webapp/script.js

```diff
@@ -49,7 +51,7 @@ async function displayComments() {
 
   commentsContainer.innerText = '';
 
-  for (comment of comments) {
+  for (const comment of comments) {
     commentsContainer.innerText += comment + '\n';
```

Comment:
nit: you may want to consider wrapping the comments in an element so that you can extend this to support other comment fields (e.g., author, etc) and style as well.

Reply:
Yeah, I was going to do that once I got to formatting the comments.